models/labels/balancer_v2/labels_balancer_v2_gauges_arbitrum.sql

```diff
@@ -0,0 +1,35 @@
+{{config(alias='balancer_v2_gauges_arbitrum',
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                    "sector",
+                                    "labels",
+                                    \'["jacektrocinski"]\') }}')}}
+
+SELECT
+    array('arbitrum') AS blockchain,
+    gauge.gauge AS address,
+    'arb:' || pools.name AS name,
+    'balancer_v2_gauges' AS category,
+    'balancerlabs' AS contributor,
+    'query' AS source,
+    TIMESTAMP('2022-01-13') AS created_at,
+    NOW() AS updated_at
```

Comment:
you'll need to add two lines for `model_name` which should just be `balancer_v2_gauges` and `label_type` as `identifier`